### DISCUSSION TITLE
Feature Request - Set default html width with yaml setting

### DISCUSSION THREAD

**Discussion by threadingdata** [quarto-dev/quarto-cli#3664] [Feature Requests]
I tend to expand the body of a html document to various layout options, like ".column-page". 

```

:::{.column-page}

 <content in wide format>

:::

```
https://quarto.org/docs/authoring/article-layout.html

The downside is the title grid width remains unchanged. While the body is wide across the screen, the title, subtitle, date, etc. section inherits the default grid's width and appears mismatched to the wider body.

Is it possible to add various page width options to the current set of yaml options for page-layout; article, full, and custom?  

For instance...

```

---
title: "Adding page-layout defaults"
format:
  html:
    page-layout: column-page
---

```

**_Other options:_**

- column-body
- column-body-outset
- column-page-inset
- column-screen

It would be great to have an option to set a document to various widths through a global yaml parameter.

Thanks for your consideration.

**Comment by mcanouil** [comment]
See related topic #2509 and #3198 (possibly other discussions).
Quarto v1.3 as per @dragonstyle comment should give more control on this aspect.

**Reply by threadingdata** [reply]
Excellent, thank you!

**Reply by dragonstyle** [reply]
More documentation about how this is supported here:

https://quarto.org/docs/prerelease/1.3.html#custom-html-article-layout-widths

**Comment by mcanouil** [comment]
Closing as completed, see <https://quarto.org/docs/output-formats/page-layout.html#grid-customization>.
